mujoco_ros/test/test_plugin/CMakeLists.txt

```diff
@@ -15,5 +15,5 @@ install(TARGETS mujoco_testplugin
 )
 install(FILES
   test_plugin.xml
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/test/test_plugin
```

Comment:
The test worked without installation before. Why did you change your mind w.r.t. installation of the test plugin?

Reply:
Turns out it was being installed already, but into package's share root, which mismatched the definition in the package.xml export path. This only worked in the devel space, because the src path to the plugin xml matched the package.xml export.

The plugin test relies on mujoco_ros to load the plugin, thus it needs to a) find the plugin xml and b) load the plugin library defined in the plugin xml.
I still don't like the fact that these artifacts are being installed into the installspace, but at least this should only happen when building tests is enabled.